migration/sql-files/023-add-space-resourcetype.sql

```diff
@@ -0,0 +1,60 @@
+-- create RESOURCE_TYPE for a SPACE type
+
+INSERT INTO resource_type 
+            (resource_type_id, 
+             NAME, 
+             created_at) 
+VALUES     ('6422fda4-a0fa-4d3c-8b79-8061e5c05e12', 
+            'openshift.io/resource/space', 
+            Now()); 
+
+
+-- create a role named 'collaborator'
+
+INSERT INTO role 
```

Comment:
I am concerned about the auth team deciding which production-level roles and scopes we are adding to the database without first discussing with other stakeholders - perhaps we could hold off on this for now?

Reply:
I understand the concern, @sbryzak . 
This is meant to replace the keycloak-authz being used in the existing Add/List Collaborator API.

Irrespective of what the discussion goes with PM, we are gonna have to retain the same behavior for a while - and these changes will help to do so.